pkg/reportxml/reportxml.go

```diff
@@ -0,0 +1,327 @@
+package reportxml
+
+import (
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
+)
+
+type (
+	// TestSuite represents formatted test suite.
+	TestSuite struct {
+		XMLName    xml.Name   `xml:"testsuite"`
+		Name       string     `xml:"name,attr"`
+		Tests      int        `xml:"tests,attr"`
```

Comment:
TestsCount ?

Reply:
This is standard Junit report format. I can not change it